packages/slate-react/src/components/editable.tsx

```diff
@@ -772,6 +773,7 @@ export const Editable = (props: EditableProps) => {
                   !IS_FIREFOX_LEGACY &&
                   !IS_IOS &&
                   !IS_QQBROWSER &&
+                  !IS_HYPERCLASS &&
```

Comment:
The electron app should also has `IS_HYPERCLASS = true`, is there any problem with it?

Reply:
@xuanduc987 what do you mean?

Reply:
```
                // COMPAT: In Chrome, `beforeinput` events for compositions
                // aren't correct and never fire the "insertFromComposition"
                // type that we need. So instead, insert whenever a composition
                // ends since it will already have been committed to the DOM.
```
is written above. As far as I understand, the code check for browser not safari, firefox, ios... and then trigger insert manually.

With this change, the hyperclass app will not trigger insert, which is good for mobile app but for electron which is a wrapper around chromium, I think the manual insert is needed.

Reply:
@xuanduc987 I see, we should apply this only to Android and iOS

Reply:
Double check on dpj issue thread https://github.com/sonyged/dpj/issues/1611

I think this fix is for problem 6 only, which is only occurs on iOS device.

With this fix, there will be problem on electron app and android app also I think, because android is also using chrome engine.

Reply:
For reference, the useragent for ios app is `Mozilla/5.0 (iPhone; CPU iPhone OS 15_0_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) HyperClass-ios-1.0.1`

Reply:
> Which should make IS_IOS true

Right. For iOS App (and for problem 6), this path never comes so problem should not be here

Reply:
@nguyenmanh1507 What do you think about this?

Reply:
To summarize the discussion, as current code,
- this path never comes for electron and Android (former works well, latter doesn't)
- iOS looks to come this path, which doesn't work well also.

Reply:
Discussion in Slack
```
manhnt  18 hours ago
This bug happen on ipad not iphone because user agent on ipad is Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) HyperClass-ios-1.0.2

manhnt  18 hours ago
Sorry I mean this will fix bug on iPadOS not iOS

esksak  15 hours ago
I see. so we should narrow down this only include HyperClass iOS/Android.
```